docs/examples/notebooks/ShapeFactor.ipynb

```diff
@@ -106,7 +106,7 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "INFO:pyhf.pdf:Validating spec against schema: /home/jovyan/pyhf/src/pyhf/data/spec.json\n",
+      "INFO:pyhf.pdf:Validating spec against schema: model.json\n",
```

Comment:
```suggestion
      "INFO:pyhf.pdf:Validating spec against schema: /home/jovyan/pyhf/src/pyhf/data/model.json\n",
```

Small note to keep this the same if you're running it from where the docs build

Reply:
hrmm, this I'm not sure why it was the full path before, but we don't store it in `data` anymore. In fact, the corresponding log statement will only print what the user specified which by default is `model.json` and the utilities will figure out the full path based on it (if one is not provided).

We store the schemas in `src/pyhf/schemas/1.0.0/` but that won't be referenced since we also bake in versioning too.

Reply:
Ah okay! Great.